src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -8,46 +8,46 @@
       "Size": 22488
     },
     "lib/arm64-v8a/lib__Microsoft.Android.Resource.Designer.dll.so": {
-      "Size": 1114
+      "Size": 18208
     },
     "lib/arm64-v8a/lib_Java.Interop.dll.so": {
-      "Size": 69368
+      "Size": 86456
     },
     "lib/arm64-v8a/lib_Mono.Android.dll.so": {
-      "Size": 98691
+      "Size": 115720
     },
```

Comment:
Why are these so much larger now?

Should we actually remove settings like `AndroidUseAssemblyStore=false` and test the defaults instead?

https://github.com/dotnet/android/blob/0297b5c92433697231218a7fd1f4dfa18d8d13d1/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest2.cs#L157

Reply:
They are larger because they not only have the real ELF header and sections, they are also aligned to the 16k boundary between sections.

A far as defaults are concerned, there are other tests which use them, so I think it's fine to leave the assembly store disabled here.